src/Tasks/Common/Resources/Strings.resx

```diff
@@ -361,4 +361,7 @@
   <data name="PackageNotFound" xml:space="preserve">
     <value>Package {0}, version {1} was not found. It might have been deleted since NuGet restore. Otherwise, NuGet restore might have only partially completed, which might have been due to maximum path length restrictions.</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="CannotFindApphostForRid" xml:space="preserve">
+    <value>Cannot find Apphost for {0}. {0} could be an invalid RuntimeIdentifier</value>
```

Comment:
@mairaw could you help us review this error message while @KathleenDollard is in Europe?

This is the error message shown when (most likely) users used a wrong runtime identifier like win-x63. Or it is a rid that is not supported

Reply:
My comments @wli3:
- should Apphost really be one word?
- RuntimeIdentifier -> runtime identifier (RID)
- missing punctuation in the last sentence
- should we add a fwlink to the RID topic for more info? https://docs.microsoft.com/dotnet/core/rid-catalog

Reply:
Will, is it practical to search our error/warning messages for Apphost to see how we usually do it. I would like consistency on that. Apphost, AppHost, or application host all make sense to me, although app host not as much. 

Same with RID vs RuntimeIdentifier. Do you have thoughts on the best place to keep notes that will help us be consistent on stuff like this?

A link to valid RIDs would be great here.

Reply:
@mairaw  @KathleenDollard 

> Apphost

according to https://docs.microsoft.com/en-us/dotnet/core/, it should be "app host". As in `The 'dotnet' app host, which is used to launch .NET Core apps. It selects the runtime and hosts the runtime, provides an assembly loading policy and launches the app. The same host is also used to launch SDK tools in much the same way.`

> should we add a fwlink to the RID topic for more info?  

created https://aka.ms/rid-catalog


> is it practical to search our error/warning messages for Apphost to see how we usually do it. I would like consistency on that. Apphost, AppHost, or application host all make sense to me, although app host not as much.

should be "app host". That doc is kind of a "front page" of dotnet core

> Same with RID vs RuntimeIdentifier. Do you have thoughts on the best place to keep notes that will help us be consistent on stuff like this?

"runtime identifier (RID)" should be the standard. As in docs.microsoft.com

# so what about that

`Cannot find app host for {0}. {0} could be an invalid runtime identifier (RID). For more information about RID https://aka.ms/rid-catalog.`

Reply:
Thanks, I prefer your version. And I added you to the aka.ms link

Reply:
Fixed, changed to discussed